reana_commons/utils.py

```diff
@@ -54,10 +54,13 @@ def click_table_printer(headers, _filter, data):
     formatted_output = "   ".join(formatted_output_parts)
     # Print the table with the headers capitalized
     click.echo(formatted_output.format(*[h.upper() for h in headers]))
-    for row in data:
+    for i, row in enumerate(data):
         if header_indexes:
             row = [row[i] for i in header_indexes]
-        click.echo(formatted_output.format(*row))
+        click.secho(
+            formatted_output.format(*row),
+            fg=colours[i] if colours and len(colours) == len(data) else None,
```

Comment:
Very minor detail which could be ignored: `len(colours) == len(data)` check could be done outside the loop to avoid the calculation on every row. In the top of the loop something like this could be done: `colours = colours if len(colours) == len(data) else None` and then `fg=colours[i] if colours else None`